samples/deploy_and_test.py

```diff
@@ -338,7 +338,7 @@ def test_sayHello_ias(self):
             xsappname = self.get_deployed_app().get_credentials_property('xsappname')
             self.assertRegex(resp.body, xsappname, 'Expected to find xsappname in response')
         else:
-            logging.warning('test_sayHello_ias was skipped. Enable it with USER_INPUT_ENABLED=true.')
+            logging.warning('test_sayHello_ias was skipped. To run test enable environment variable USER_INPUT_ENABLED=true and enter "y" for the user prompt: "Do you want to add role manually?."')
```

Comment:
I see... is this question really required or is it enough to just say, "please assign role-collection and if done... press enter..."

Reply:
actually now that we have the env variable it is somewhat redundant. Good point! 💯